Source/OxyPlot/Graphics/Snapping.cs

```diff
@@ -0,0 +1,39 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="Snapping.cs" company="OxyPlot">
+//   Copyright (c) 2014 OxyPlot contributors
+// </copyright>
+// <summary>
+//   Snapping data.
+// </summary>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot
+{
+    /// <summary>
+    /// Snapping data.
+    /// </summary>
+    public class Snapping
```

Comment:
The reason I used an object is not to cause breaking changes in the future. If you want new snapping data in the future, you can add it without a breaking change or making a mess. But happy to make it direct properties (they were direct properties in the first place).